openupgrade_scripts/scripts/account/14.0.1.1/upgrade_analysis.txt

```diff
@@ -362,6 +364,11 @@ DEL ir.model.access: account.access_account_tax_report_line_invoice
 DEL ir.model.access: account_analytic_default.access_account_analytic_default [renamed to account module]
 DEL ir.model.access: account_analytic_default.access_account_analytic_default_analytic [renamed to account module]
 DEL ir.model.access: account_analytic_default.access_account_analytic_default_invoice [renamed to account module]
+NEW ir.model.constraint: account.constraint_account_group_check_length_prefix
+ir.model.constraint: account.constraint_account_move_line_check_amount_currency_balance_sign (changed definition: is now 'check(   (    (currency_id != company_currency_id)    and    (    (debit - credit <= 0 and amount_currency <= 0)     or    (debit - credit >= 0 and amount_currency >= 0)    )    )    or   (    currency_id = company_currency_id    and    round(debit - credit - amount_currency,2) = 0    )   )' ('check(    currency_id is null    or    company_currency_id is null    or   (    (currency_id != company_currency_id)    and    (    (balance > 0 and amount_currency > 0)     or(balance <= 0 and amount_currency <= 0)     or(balance >= 0 and amount_currency >= 0)    )    )   )'))
```

Comment:
This one requires any check?

Reply:
odoo automatically update the changed constraints

Reply:
I mean that the data doesn't comply with this new check.

Reply:
Well, if it doesn't comply, then the constraint won't be added.

Reply:
I think this crashes in such case. The question is if due to the migration, any data may not comply this requirement.

Reply:
Right. Then, I can delete the constraint and then wait to do trying to add the new version of this constraint. And i can't. then no problem.